gamesinfo/skyrimspecialedition.sh

```diff
@@ -6,10 +6,10 @@ game_proton_options="--native 'xaudio2_7' -e 'PULSE_LATENCY_MSEC=90'"
 game_wine_options="--native 'xaudio2_7'"
 game_protontricks=""
 game_winetricks=""
-game_scriptextender_url="https://skse.silverlock.org/beta/skse64_2_01_04.7z"
+game_scriptextender_url="https://skse.silverlock.org/beta/skse64_2_01_05.7z"
 game_scriptextender_files=( \
 	"skse64_2_01_04/Data" \
-	"skse64_2_01_04/skse64_1_6_342.dll" \
+	"skse64_2_01_04/skse64_1_6_353.dll" \
 	"skse64_2_01_04/skse64_loader.exe" \
 	"skse64_2_01_04/skse64_steam_loader.dll" \
```

Comment:
You also need to change the directories from `skse64_2_01_04` to `skse64_2_01_05`